tests/Integration/Behaviour/Features/Context/Domain/Carrier/CarrierRangesFeatureContext.php

```diff
@@ -148,20 +146,6 @@ private function getCarrierRanges(string $reference, ShopConstraint $shopConstra
             $rangesExpected = new CarrierRangesCollection($data);
 
             Assert::assertEquals($rangesExpected, $rangesDatabase);
-
-            // Automatically checks that the carrier_zone association is properly set
```

Comment:
Ok for the removal of this automatic check But then we need to add assertions in the `carrier_ranges.feature` The set ranges command can also update the associated zones based on the provided ranges by zones

We must make sure that it updates the zones appropriately, the good news is that you can now check it manually and explicitly in the gerkhin since the list of zones is not returned by the CQRS query